CITATION

```diff
@@ -0,0 +1,18 @@
+@inproceedings{Chen:2016:XST:2939672.2939785,
+ author = {Chen, Tianqi and Guestrin, Carlos},
+ title = {XGBoost: A Scalable Tree Boosting System},
+ booktitle = {Proceedings of the 22Nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining},
```

Comment:
Change it to
```
booktitle = {Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining},
```

Reply:
I didn't type this, but fetched it from "Export Formats: BibTeX" on https://dl.acm.org/citation.cfm?id=2939785, so it might need to be changed there officially as well.